docroot/themes/humsci/humsci_basic/src/scss/components/_not-layout-builder-main-content.scss

```diff
@@ -1,3 +1,5 @@
+// WHAT FORM STYLES ARE THESE FOR??
```

Comment:
@bryanbraun do you know what this is used for?

Reply:
update, from Heather:

> The search results page doesn't use layout builder. We needed to add a class to pages that don't use layout builder so that we could styles elements on it.